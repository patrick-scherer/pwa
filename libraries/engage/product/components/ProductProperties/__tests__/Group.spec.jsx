import React from 'react';
import { mount } from 'enzyme';
import Group from '../Group';

describe('<Group />', () => {
  it('should render a group title row', () => {
    const wrapper = mount(<Group group="Test" />);
    expect(wrapper.find('tr').length).toEqual(1);
    expect(wrapper.find('td').length).toEqual(1);
    expect(wrapper.find('td').text()).toEqual('Test');
    expect(wrapper).toMatchSnapshot();
  });
});
