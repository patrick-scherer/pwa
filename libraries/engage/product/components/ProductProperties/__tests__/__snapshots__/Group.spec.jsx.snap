// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Group /> should render a group title row 1`] = `
<Group
  group="Test"
>
  <tr>
    <td
      className={
        Object {
          "data-css-1o9kcbq": "",
        }
      }
      colSpan="2"
    >
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": "Test",
          }
        }
      />
    </td>
  </tr>
</Group>
`;
